Assets/Tests/TestBulletPatterns.cs

```diff
@@ -1,17 +1,19 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using BulletManagement;
 using NUnit.Framework;
 using UnityEngine;
 using UnityEngine.TestTools;
 
 namespace Tests
 {
+    [TestFixture]
     public class TestBulletPatterns
     {
-        public Vector3[] getBulletPositions(Bullet[] bullets)
+        private static Vector3[] getBulletPositions(IReadOnlyList<Bullet> bullets)
```

Comment:
Can you link the documentation for where you found this fixture? Thanks.

Reply:
Fixture for class, sorry wrong location.

Reply:
https://docs.nunit.org/articles/nunit/writing-tests/attributes/testfixture.html

Reply:
Sweet, thanks.